lib/hooks/useFetchAllContributors.ts

```diff
@@ -49,15 +49,15 @@ const useFetchAllContributors = (query: queryObj, config?: SWRConfiguration) =>
   const endpointString = `${baseEndpoint}?${urlQuery}`;
 
   const { data, error, mutate } = useSWR<PaginatedResponse, Error>(
-    endpointString,
+    makeRequest ? endpointString : null,
     publicApiFetcher as Fetcher<PaginatedResponse, Error>,
     config
   );
 
   return {
     data: data?.data ?? [],
     meta: data?.meta ?? { itemCount: 0, limit: 0, page: 0, hasNextPage: false, hasPreviousPage: false, pageCount: 0 },
-    isLoading: !error && !data,
+    isLoading: !error && !data && makeRequest,
```

Comment:
It's because you can't have conditional hooks and when the user is not searching for anything we don't want to show the first page of data of potential contributors to add. We want to show the empty state (magnifying glass SVG).

Definitely open to suggestions. I had tried a null URL with SWC, but that didn't work.

Reply:
I think longer term for the explore contributors in general, we could have it only load data if filters are applied. I didn't make that change as the grid is used elsewhere atm.